x-pack/plugins/cases/public/components/user_action_tree/helpers.test.tsx

```diff
@@ -142,50 +142,74 @@ describe('User action tree helpers', () => {
       `${i18n.UPDATE_INCIDENT} ${basicPush.connectorName}`
     );
     expect(wrapper.find(`[data-test-subj="pushed-value"]`).first().prop('href')).toEqual(
-      JSON.parse(action.newValue).external_url
+      JSON.parse(action.newValue!).external_url
     );
   });
 
-  it('label title generated for update connector - change connector', () => {
-    const action = {
-      ...getUserAction(['connector'], 'update'),
-      oldValue: JSON.stringify({ id: 'servicenow-1' }),
-      newValue: JSON.stringify({ id: 'resilient-2' }),
-    };
-    const result: string | JSX.Element = getConnectorLabelTitle({
-      action,
-      connectors,
-    });
-
-    expect(result).toEqual('selected My Connector 2 as incident management system');
-  });
-
-  it('label title generated for update connector - change connector to none', () => {
-    const action = {
-      ...getUserAction(['connector'], 'update'),
-      oldValue: JSON.stringify({ id: 'servicenow-1' }),
-      newValue: JSON.stringify({ id: 'none' }),
-    };
-    const result: string | JSX.Element = getConnectorLabelTitle({
-      action,
-      connectors,
+  describe('getConnectorLabelTitle', () => {
```

Comment:
Most of these changes were from wrapping the `getConnectorLabelTitle` in a `describe` block but I also had to make the `oldValue` and `newValue` fields valid connectors so they get parsed correctly.